modules/troubleshooting/pages/releases.adoc

```diff
@@ -1,3 +1,13 @@
 = Troubleshooting Releases
 
+== Release fails because of signature check
+
+- Check if push pipeline has `chains.tekton.dev/signed=failed` annotation.
+- That means that it failed on chains.
+- If chains is failing with `POST https://quay.io/v2/image-path/blobs/uploads/: UNAUTHORIZED: access to the requested resource is not authorized; map[]`.
+- Since build was successful, problem is with service account `appstudio-pipeline` which has
+  in `imagePullSecrets` section some wrongly added pull secrets for registry.
+- In imagePullSecrets should be only `appstudio-pipeline-dockercfg-*` secret which provides access to
+  pipeline/task bundles.
+
 include::partial${context}-releases.adoc[]
```

Comment:
We need to remove the content of this partial. It adds a line to the bottom of the page indicating that there is no information for releases yet.

Reply:
removed